components/notebook-controller/pkg/controller/notebook/notebook_controller.go

```diff
@@ -438,7 +438,8 @@ func generateVirtualService(instance *v1alpha1.Notebook) (*unstructured.Unstruct
 	if err := unstructured.SetNestedStringSlice(vsvc.Object, []string{"*"}, "spec", "hosts"); err != nil {
 		return nil, fmt.Errorf("Set .spec.hosts error: %v", err)
 	}
-	if err := unstructured.SetNestedStringSlice(vsvc.Object, []string{"kubeflow-gateway"},
+	// TODO(kunming): change to <namespace>/<name> format once Istio update to 1.1 stable release.
```

Comment:
we have 1.1 now, is namespace/name working?

Reply:
Nice